components/02-elements/tab/_tab.scss

```diff
@@ -67,14 +73,13 @@ $tab__icon-size                         : 48px !default;
                 border-bottom: $tab__title-border--large-active;
                 background-color: $tab__title-background--large-active;
                 font-weight: $tab__title-font-weight--large-active;
-                color: $tab__title-color;
                 z-index: 2;
             }
             .tab__icon {
                 transform: rotateX(180deg);
 
                 svg {
-                    fill: $color-primary;
+                    fill: $tab__icon-color--active;
```

Comment:
Please add BEM class

Reply:
It's different component I was just fixing color values. Of course I can change but I guess it passed some CR before and was ok ?

Reply:
done plus some other refactoring in tab as I saw more things to improve